telemetry/telemetry.go

```diff
@@ -114,7 +114,7 @@ func getDistinctID() string {
 	// Otherwise use the configured API Key.
 	// Failing that, try to use the user name and host name?
 
-	id := os.Getenv("AKITA_SEGMENT_DISTINCT_ID")
+	id := os.Getenv("POSTMAN_ANALYTICS_DISTINCT_ID")
```

Comment:
Do we need to do any changes for this in documentations?
cc @mgritter @liujed

Reply:
No, this is not a documented feature.  The ability to disable analytics is more important.